### PR TITLE
cueobserve#158 

### PR DESCRIPTION
**Pull request by kshitij-cuebook**
1) docker-compose-dev created for faster local development, 2) redis converted to a service, 3) environment variables are not automatically used in docker-compose, 4) unnecessary files removed

### CODE REVIEW

File: api/start_server.sh [hunk 1]
```diff
@@ -10,13 +9,11 @@ fi
...
+mkdir -p /home/staticfiles
 mv static_root/* /home/staticfiles
 rm -rf static_root/
-(/usr/bin/redis-server) &
-(gunicorn app.wsgi --user www-data --bind 0.0.0.0:8000 --workers 3 --timeout 300) &
 (celery -A app worker --concurrency=2 -l INFO --purge) &
 (celery -A app worker --concurrency=4 -Q anomalySubTask -l INFO --purge) &
 (celery -A app beat -l INFO --scheduler django_celery_beat.schedulers:DatabaseScheduler) &
-echo "starting nginx"
-nginx -g "daemon off;"
```
Comment (vincue) [inline api/start_server.sh:22]: This file is being used in current deployment. Don't update it unless deployment for production is updated.

File: docker-compose.yml [hunk 2]
```diff
@@ -20,11 +33,9 @@ services:
     build:
       context: ui
       dockerfile: Dockerfile
-    volumes:
-      - /frontend:/app
     network_mode: "host"
 
-  postgresql-cue:
+  cueo-postgresql:
```
Comment (vincue) [inline docker-compose.yml:38]: Postgres is not needed in docker-compose, sqllite will be used in that scenario.

File: docker-compose.yml [hunk 3]
```diff
@@ -20,11 +33,9 @@ services:
     build:
       context: ui
       dockerfile: Dockerfile
-    volumes:
-      - /frontend:/app
     network_mode: "host"
```
Comment (vincue) [inline docker-compose.yml:36]: Now if we have separate docker-compose for dev and production, network_mode host should be removed and we should replace build context with deployed image. I don't think it's convenient that user has to build images.

Comment (kshitij-cuebook) [reply docker-compose.yml:36]: We can do these changes in docker-compose in a future PR.

File: docker-compose.yml (hunk 2 above)
Comment (kshitij-cuebook) [reply docker-compose.yml:38]: As discussed, let's just keep this in docker-compose-dev and remove it from docker-compose (later)

File: api/start_server.sh (hunk 1 above)
Comment (kshitij-cuebook) [reply api/start_server.sh]: I have I have removed this file's changes.

**Pull request merged**